pylal/coh_PTF_pyutils.py

```diff
@@ -85,7 +85,7 @@ def get_bestnr( trig, q=4.0, n=3.0, null_thresh=(4.25,6), snr_threshold=6.,\
         sens[ifo] = getattr(trig, 'sigmasq_%s' % i.lower()) * \
                         sum(numpy.array([fPlus[ifo], fCross[ifo]])**2)
     ifos.sort(key=lambda ifo: sens[ifo], reverse=True)
-    for i in xrange(0,2):
+    for i in xrange(0,len(ifos)):
```

Comment:
Ahh, I think I understand now that this is changed for single-detector running. In that case I would simply not run this single detector SNR test at all. The behaviour for len(ifos) > 2 should not be changed unless it is demonstrated to be better though!